config/initializers/redis.rb

```diff
@@ -0,0 +1,3 @@
+# Prevents error messages in sidekiq in dev boxes.
+# should be removed once sidekiq uses #exists?
```

Comment:
This is a good move, that error message is already bugging me. Maybe add a TODO here to remind us to remove it? It's likely to get forgotten about otherwise